File: IT/Eccezioni/Esercizi/C++/esercizio3.cpp
/* file esercizio3.cpp */

#include <iostream>
#include <stdexcept>

using std::cout;
using std::endl;
using std::string;
using std::out_of_range;

int main()
{
	string var = "Cantami, o Diva, del pelide Achille l'ira funesta";

	int i = 0;

	try {
		while (true) {
			cout << var.at(i);
			i++;
		}
	}
	catch (out_of_range& e) {
		cout << endl;
	}

	cout << "Fine del programma." << endl;
}

File: IT/Eccezioni/Esercizi/Python/esercizio3.py
# file esercizio3.py

from __future__ import print_function
from sys import stdout
from os import linesep

var = "Cantami, o Diva, del pelide Achille l'ira funesta"

try:
	i = 0
	while (True):
		print(var[i], end="")
		i += 1

except IndexError:
	stdout.write(linesep)

print("Fine del programma.")

